ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-eus-4.10.yaml

```diff
@@ -99,6 +98,18 @@ tests:
       UPGRADE_PRE_RUN_TAGS: '@arm64 and @upgrade-prepare and @4.10 and @aws-ipi and
         not @fips'
     test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_ARM64_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
```

Comment:
to keep consistent, shall we use `RELEASE_IMAGE_ARM64_INTERMEDIATE`?

Reply:
updated.